### PR TITLE
CAS-475 FE implement the results page for bed space with improved and increased details

### PR DESCRIPTION
**Pull request by aliuk2012**
# Context
https://dsdmoj.atlassian.net/browse/CAS-475

# Changes in this PR

- Updated the copy of the page heading
- Added new h2 as described in the tickets description
- Update “Bedspaces” in the summary cards to show how many beds in the property are booked and available.

### Before
image

### After
image

# Release checklist

[Release process documentation](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/edit-v2/4247847062?draftShareId=a1c360ab-bd31-4db1-aae3-7cc002761de9).

## Pre-merge checklist

- [ ] Are any changes required to dependent services for this change to work?
  (eg. CAS API)
- [ ] Have they been released to production already?

## Post-merge checklist

- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to test
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to preprod
- [ ] [Manually approve](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process#Manual-releases)
  release to prod

### CODE REVIEW

File: server/views/temporary-accommodation/bedspace-search/results.njk [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 {% extends "../../partials/layout.njk" %}
 
-{% set pageTitle = "Bedspace search results (" + results.length + ") - " + applicationName %}
+{% set pageTitle = "Search for available bedspaces - " + results.length + "results - " + applicationName %}
```
Comment (froddd) [inline server/views/temporary-accommodation/bedspace-search/results.njk]: Ideally should handle '1 result' with no inflection

File: server/views/temporary-accommodation/bedspace-search/results.njk [hunk 2]
```diff
@@ -17,7 +17,9 @@
 {% block content %}
     {% include "../../_messages.njk" %}
 
-    <h1 class="govuk-heading-l">Bedspace search results ({{ results.length }})</h1>
+    <h1 class="govuk-heading-l">Search for available bedspaces</h1>
+
+    <h2 class="govuk-heading-m">{{ results.length }} results for your bedspace search</h2>
```
Comment (froddd) [inline server/views/temporary-accommodation/bedspace-search/results.njk]: Again, should handle '1 result' with no inflection

**Review by froddd — COMMENTED** [review]
For completeness, should handle case where there's only 1 result -- at the moment anything with the number of results would show the word 'results' as plural.

**Comment by aliuk2012** [comment]
Closing this as there is an issue with existing data not prepopulating the new fields. Will be looked as again as part of https://dsdmoj.atlassian.net/browse/CAS-995

**Review by froddd — APPROVED** [review]
👍

**Pull request merged**